modules/blockuser.cpp

```diff
@@ -58,7 +58,8 @@ class CBlockUser : public CModule {
 
 	EModRet OnLoginAttempt(std::shared_ptr<CAuthBase> Auth) override {
 		if (IsBlocked(Auth->GetUsername())) {
-			Auth->RefuseLogin(MESSAGE);
+			CString sMessage = GetNV(Auth->GetUsername());
+			Auth->RefuseLogin(sMessage.empty() ? DEFAULT_MESSAGE : sMessage);
```

Comment:
```Auth->RefuseLogin("Blocked: " + sMessage.empty() ? DEFAULT_MESSAGE : sMessage));``` would make this post the error message "Blocked: REASON", such that if no input is provided, then it'd say "Blocked: Your account has been disabled. Contact your administrator."

This works a little better for custom ban reasons, but this is just my opinion.